.github/workflows/release-code-freeze.yml

```diff
@@ -41,10 +41,24 @@ jobs:
               with:
                   fetch-depth: 0
 
+            - name: Setup PNPM
```

Comment:
Since this job used to be just calling an action, no checkout/build was needed. this job is isolated from the others so needs its own setup now. In future we can either merge this back into the code freeze job or a lot of duplicate checkout and build won't be needed if we publish the CLI as a package.